File: MobileApp/Client/MyHome/app/src/main/java/com/example/myhome/House.java
package com.example.myhome;

public class House {
    private int HouseID;
    private int AlarmStatus;
    private String OwnerID;
    private Address HouseAddress;

    public int getHouseID() {
        return HouseID;
    }

    public void setHouseID(int houseID) {
        HouseID = houseID;
    }

    public int getAlarmStatus() {
        return AlarmStatus;
    }

    public void setAlarmStatus(int alarmStatus) {
        AlarmStatus = alarmStatus;
    }

    public String getOwnerID() {
        return OwnerID;
    }

    public void setOwnerID(String ownerID) {
        OwnerID = ownerID;
    }

    public Address getHouseAddress() {
        return HouseAddress;
    }

    public void setHouseAddress(Address houseAddress) {
        HouseAddress = houseAddress;
    }
}

File: Service/LiveSSH/Models/House.cs
using SSH_Service.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace SSH_Service
{
    public class House
    {
        public int HouseID { get; set; }
        public int AlarmStatus { get; set; }
        public string OwnerID { get; set; }
        public Address HouseAddress { get; set; }
    }
}